libdevcore/RangeMask.h

```diff
@@ -94,8 +94,8 @@ class RangeMask
 	/// reference to itself.
 	RangeMask& invert() { return *this = inverted(); }
 
-	template <class S> RangeMask operator-(S const& _m) const { auto ret = *this; return ret -= _m; }
-	template <class S> RangeMask& operator-=(S const& _m) { return invert().unionWith(_m).invert(); }
+	RangeMask operator-(RangeMask const& _m) const { auto ret = *this; return ret -= _m; }
```

Comment:
Looks like we won't be able to do `-` and `-=` with a single number after this... Maybe not a problem, but one convenient usage less